src/main/java/de/tum/in/www1/artemis/service/iris/IrisSessionService.java

```diff
@@ -58,11 +65,15 @@ public void checkIsIrisActivated(IrisSession session) {
      *
      * @param session The session to check
      * @param user    The user to check
+     * @throws AccessForbiddenException If the user has not accepted the Iris privacy policy yet
      */
     public void checkHasAccessToIrisSession(IrisSession session, User user) {
```

Comment:
I'd like to point out that the whole PR (or even Iris Sub-System) is lacking any annotations like `@Nullable` or `@NotNull`. Maybe we should add them in the future. Especially in this case it's not obvious, that the current user is fetched from the DB if the input is `null`.

Reply:
It is explained in the javadoc but you're right it should be there